CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.14.0 (2021-05-26)
+
+- Updated the lookup component to collapse tokens when more than 5 items are selected and the picker to show more results is enabled. [#186](https://github.com/blackbaud/skyux-lookup/pull/186)
```

Comment:
```suggestion
- Updated the lookup component to collapse tokens if users select more than 5 items when the lookup menu includes the 'Show all/matches' button. [#186](https://github.com/blackbaud/skyux-lookup/pull/186)
```
Wasn't sure at first what we meant by "and the picker to show more results is enabled," but I _think_ we mean something like this. I used terminology from Adam's design guidelines for consistency with the pending docs.